snippets/base/views.py

```diff
@@ -24,7 +24,8 @@
 from snippets.base.util import get_object_or_none
 
 
-_http_max_age = lambda: getattr(settings, 'SNIPPET_HTTP_MAX_AGE', 90)
+def _http_max_age():
+    return getattr(settings, 'SNIPPET_HTTP_MAX_AGE', 90)
```

Comment:
quoting pep8  `Separate top-level function and class definitions with two blank lines.` It doesn't say anything about seperating top-level functions and variables, so I believe that's OK. 

https://www.python.org/dev/peps/pep-0008/#blank-lines